tests/jenkins/Jenkinsfile

```
@@ -328,6 +373,25 @@ pipeline {
                         }
                     }
                 }
+                stage('tqperf'){
+                    stages{
+                        stage('neuron'){
+                            steps{
+                                sh 'sh tests/jenkins/run_neuron.sh tqperf "" 8'
```

```suggestion
                                sh 'sh tests/jenkins/run_neuron.sh tqperf tqperf 6'
```
Jenkins nodes have 6 ranks, so there is no reason to use 8 here and oversubscribe 2 ranks